README.md

```diff
@@ -207,4 +207,23 @@ see the [Scaling Lightning](https://github.com/scaling-lightning/scaling-lightni
 project.
 
 ## Docker
-If you want to run the cli in a containerized environment, see the docker set up docs [here](./docker/README.md)
\ No newline at end of file
+If you want to run the cli in a containerized environment, see the docker set up docs [here](./docker/README.md)
+
+## Contributing
```

Comment:
Thanks for adding this! 

Let's pull it out into `CONTRIBUTING.md` and link to it from `README`? We'll probably end up pulling it out eventually so might as well start like that :)